src/index.tsx

```diff
@@ -13,12 +16,24 @@ if (process.env.NODE_ENV === 'development') {
   });
 }
 
-ReactDOM.render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
-  document.getElementById('root')
-);
+if (typeof (window as any).TextEncoder === 'undefined') {
+  (window as any).TextEncoder = TextEncoder;
+}
+
+let app;
+const browser: any = detect();
+console.log(browser.name);
```

Comment:
we should perhaps remove this

Reply:
I will correct and merge after the freeze ends